flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/snapshot/RocksNativeFullSnapshotStrategy.java

```diff
@@ -219,34 +219,40 @@ public SnapshotResult<KeyedStateHandle> get(CloseableRegistry snapshotCloseableR
             }
         }
 
-        private void uploadSstFiles(
-                @Nonnull Map<StateHandleID, StreamStateHandle> privateFiles,
+        /** upload files and return total uploaded size. */
+        private long uploadSnapshotFiles(
+                @Nonnull List<HandleAndLocalPath> privateFiles,
```

Comment:
NIT: This method can be simplified by returned the list of files and computing the size in the caller.
Not sure if this is in scope of this PR or not.

Reply:
Caller is more complex, maybe it's better to keep the status quo?